### DISCUSSION TITLE
Twig 3?

### DISCUSSION THREAD

**Discussion by mledwards** [Cecilapp/Cecil#1360] [Ideas]
Just wondering if there were any plans to upgrade Twig 2 to Twig 3? Porting a project to Cecil, and finding some issues in the way Twig 2 and 3 deal with imports outside macros.

**Comment by ArnaudLigny** [comment]
Hello @mledwards, Twig have been upgraded to 3.x since version 6.0.0 of Cecil (+ support of PHP 8).
